tests/utils/test_hook.py

```diff
@@ -0,0 +1,95 @@
+import mock
+
+from ddtrace.compat import PY2
+from ddtrace.utils.hook import (
+    register_post_import_hook,
+    deregister_post_import_hook,
+)
+
+from tests.subprocesstest import SubprocessTestCase, run_in_subprocess
+
+
+if PY2:
+    reload = reload
+else:
+    import importlib
+    reload = importlib.reload
```

Comment:
please move this into `compat.py` and make it be `reload_module` (the name used by `six`)